.storybook/addons/props/About.js

```diff
@@ -57,7 +75,20 @@ export default class About extends React.Component {
           </Description>
         )}
 
-        {props.length > 0 && <PropTable props={props} />}
+        {changelog.length > 0 && (
+          <>
+            <SubHeader>Past 1 month</SubHeader>
+            <Header>Changelog</Header>
+            <LogTable logs={changelog} />
```

Comment:
i think the change log table should go after props/go last from an information arch perspective and order of importance

Reply:
I had the same thought at first, but in reality, you would never see the changelog otherwise. And since the changelog is only for the past month of commits, this log table isn't going to be very large.

Reply:
@stefhatcher Bump :)

Reply:
oooh, okay. that makes sense too.